dev/tools/gen_defaults/data/checkbox.json

```diff
@@ -3,6 +3,7 @@
 
   "md.comp.checkbox.container.height": 18.0,
   "md.comp.checkbox.container.width": 18.0,
+  "md.comp.checkbox.container.shape": 2.0,
```

Comment:
these files are generated and shouldn't be modified (sent you a link), consider using the existing `md.comp.checkbox.unselected.outline.width`

Reply:
Thanks - done